cordformation/src/main/kotlin/net/corda/plugins/Node.kt

```diff
@@ -292,6 +292,9 @@ open class Node @Inject constructor(private val project: Project) {
         config.getInt("sshd.port")
     }
 
+    @get:Internal
+    internal var usesDefaultSSHPort: Boolean = false
```

Comment:
Actually, come to think of it, can you also make this property's setter function `private` please?
```kotlin
@get:Internal
internal var usesDefaultSSHPort: Boolean = false
    private set
```
I trust `private` more than I trust Kotlin's `internal` scope.